great_expectations/expectations/core/expect_column_unique_value_count_to_be_between.py

```diff
@@ -124,22 +124,23 @@ class ExpectColumnUniqueValueCountToBeBetween(ColumnExpectation):
     default_profiler_config: RuleBasedProfilerConfig = RuleBasedProfilerConfig(
         name="expect_column_unique_value_count_to_be_between",  # Convention: use "expectation_type" as profiler name.
         config_version=1.0,
-        variables={
-            "mostly": 1.0,
-            "strict_min": False,
-            "strict_max": False,
-            "false_positive_rate": 0.05,
-            "estimator": "bootstrap",
-            "num_bootstrap_samples": 9999,
-            "bootstrap_random_seed": None,
-            "truncate_values": {
-                "lower_bound": 0,
-                "upper_bound": None,
-            },
-            "round_decimals": 0,
-        },
+        variables={},
```

Comment:
For self-initializing Expectations, `variables` are moved from the `Profiler`-level to the `Rule`-level